core/streamly-core.cabal

```diff
@@ -355,6 +355,7 @@ library
                      , Streamly.Data.Fold.Tee
                      , Streamly.Data.Parser
                      , Streamly.Data.Unfold
+                     , Streamly.Data.Array
```

Comment:
We are exposing this. We might need to review the API.

Reply:
Yes, we will review. I had taken a look and it looked ok.